docs/changelog/1_0_rc1.rst

```diff
@@ -0,0 +1,139 @@
+========
+1.0 RC 1
+========
+
+This changelog summarizes new features and breaking changes in
+EdgeDB 1.0 RC 1 "Ran".
+
+
+EdgeQL
+======
+
+* Drop ``FINAL`` from the syntax (:eql:gh:`#2607`).
```

Comment:
Have we actually dropped the syntax? @msullivan

Reply:
From what I understand we no longer support it for most cases:
```
db> create final scalar type foo extending str;
error: UnsupportedFeatureError: FINAL is not supported
  ┌─ query:1:1
  │
1 │ create final scalar type foo extending str;
  │ ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ error
```
But I guess we have a way of allowing it in dumps and migrations: #2886

So as a language feature it's not longer part of any actual construct and is deprecated, but not outright an error for some legacy cases. Which I think makes it fair to say that we've "dropped" it as we don't actually want anyone to use it.